src/Symfony/Component/DependencyInjection/Argument/AbstractableArgument.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+/**
+ * Represents an abstract parameter, which have to be set by a compiler pass or a DI extension.
+ */
+final class AbstractableArgument
```

Comment:
why not AbstractArgument? - abstract-able is strange, not sure what it would mean

Reply:
it was AbstractArgument, but I thought [this comment](https://github.com/symfony/symfony/pull/35076#discussion_r361403197) was about class name and renamed it

and I couldn't think of a better name :)

Reply:
should it be renamed to AbstractArgument?

Reply:
yes please

Reply:
done